cmd/cl-dataplane/app/envoyconf.go

```diff
@@ -92,6 +92,7 @@ static_resources:
       typed_config:
         "@type": type.googleapis.com/envoy.extensions.transport_sockets.tls.v3.UpstreamTlsContext
         sni: {{.controlplaneGRPCSNI}}
+        max_session_keys: 0
```

Comment:
It would be better to have TODO remove it after the SNI proxy is removed, or maybe explain why you put it in the code.